interfaces/portal/src/app/pages/program-registration-debit-cards/components/link-card-on-site-dialog/link-card-dialog.component.ts

```diff
@@ -51,43 +46,114 @@ export class LinkCardDialogComponent {
     () => !this.tokenCode().includes('_') && this.tokenCode() !== '',
   );
 
-  readonly dialogLabels = computed(() => {
+  readonly dialogTitle = computed(() => {
+    if (this.currentTokenCode()) {
+      return $localize`Replace visa card`;
+    }
+
+    return $localize`Link visa card`;
+  });
+
+  readonly confirmationButtonLabel = computed(() => {
     if (this.currentTokenCode()) {
-      return {
-        header: $localize`Replace visa card`,
-        confirmationButton: $localize`Replace card`,
-      };
+      return $localize`Replace card`;
+    }
+
+    return $localize`Link card`;
+  });
+
+  readonly tokenCodeWithoutDashes = ({
+    tokenCodeToClean,
+  }: {
+    tokenCodeToClean: string;
+  }) => computed(() => tokenCodeToClean.replaceAll('-', ''));
```

Comment:
This is 'just a pure function', right?
Does it _need_ to be computed at all?

I don't think we need a lot of that Signal-magic-dust here...